addon/bootstrap.js

```diff
@@ -17,12 +18,16 @@ const { studyUtils } = Cu.import(STUDYUTILSPATH, {});
 
 const REASONS = studyUtils.REASONS;
 
-// QA NOTE: Study Specific Modules - package.json:addon.chromeResouce
-const BASE = `template-shield-study-button-study`;
-XPCOMUtils.defineLazyModuleGetter(this, "Feature", `resource://${BASE}/lib/Feature.jsm`);
+// logging
```

Comment:
This seems very similar/identical to studyUtils.createLog? Why not call it instead? I'm still not sure what the log is for -- perhaps explain as a comment?

Reply:
Hm. you are identifiying much that is mysterious.  I will try setting up CI soon.